Source/WebCore/css/parser/SizesCalcParser.cpp

```diff
@@ -187,12 +187,12 @@ bool SizesCalcParser::calcToReversePolishNotation(CSSParserTokenRange range)
     // When there are no more tokens to read:
     // While there are still operator tokens in the stack:
     while (!stack.isEmpty()) {
-        // If the operator token on the top of the stack is a parenthesis, then there are mismatched parentheses.
+        // If the operator token on the top of the stack is a parenthesis, then there are unclosed parentheses.
         CSSParserTokenType type = stack.last().type();
-        if (type == LeftParenthesisToken || type == FunctionToken)
-            return false;
-        // Pop the operator onto the output queue.
-        appendOperator(stack.last());
+        if (type != LeftParenthesisToken && type != FunctionToken) {
+            // Pop the operator onto the output queue.
+            appendOperator(stack.last());
+        }
```

Comment:
This seems fine to me, but I think Antti or Ryosuke (or Matthieu!) should double-check.